<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>telaCliente</class>
 <widget class="QDialog" name="telaCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>28</y>
     <width>761</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold SemiConden</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Bem vindo </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="txt_nome">
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold SemiConden</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold SemiConden</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="txt_id">
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold SemiConden</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tb_menu">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>561</width>
     <height>421</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>Teste</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Teste</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>tst</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_reservas">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>80</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Reservas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_notificacoes">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>140</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Notificações</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_avaliacoes">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>200</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Avaliações</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
